MREUnityRuntime/MREUnityRuntimeLib/Core/ActorManager.cs

```diff
@@ -86,6 +86,14 @@ internal Actor FindActor(Guid id)
             }
         }
 
+        internal void UpdateAllVisibility()
+        {
+            foreach(var actor in _actorMapping.Values.Where(a => a.Parent == null))
+            {
+                Actor.ApplyVisibilityUpdate(actor, force: true);
+            }
```

Comment:
This needs to work on actor queues instead, queueing a local command to execute once the actor is loaded.